internal/provider/resource_cluster_personality_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 Broadcom. All Rights Reserved.
+// SPDX-License-Identifier: MPL-2.0
+
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/vmware/terraform-provider-vcf/internal/constants"
+	"os"
+	"testing"
+)
+
+const (
+	VcfTestClusterId = "VCF_TEST_CLUSTER_ID"
+)
```

Comment:
Why a const here and not in the "constants" file?

Reply:
Because it's only used in one test file. "internal/constants.go" is already full of single-use constants.

But now that you've mentioned it I notice that constants.go is the only place we store environment variable names for tests. I'll move this one there too so that there are no exceptions